apps/protailwind/studio/schemas/documents/module.js

```diff
@@ -76,6 +76,19 @@ export default {
       title: 'Body',
       type: 'body',
     },
+    {
+      name: 'products',
+      title: 'Products',
+      description: 'Sellable products associated with this module',
+      type: 'array',
+      of: [
+        {
+          title: 'Exercise, Section, or Explainer',
+          type: 'reference',
+          to: [{title: 'Product', type: 'product'}],
+        },
+      ],
+    },
```

Comment:
I don't think that we need to have a reference link like this and an individual module is likely just an association with a `productId` - We already have `Product` in the Prisma Schema, so imo this should generally be a reference to that.

Another way to look at this would be to have the Product as a `resource` object (similar to `solution` in Exercise) that can be filtered for.

What's the advantage to having a top level `Product` document? I think the module has most of the metadata (image, title, etc and features is probably not a fit here)?

Reply:
I like the resource object approach and updated my PR with it. 👍